src/server/misc/Utilities.ts

```diff
@@ -5,10 +5,19 @@ import * as vm from 'vm';
 import { ICompilerFile } from '../compiler';
 
 enum AllowedInternalModules {
+    assert,
+    async_hooks,
+    buffer,
+    crypto,
+    dns,
     path,
+    querystring,
+    readline,
+    stream,
+    string_decoder,
+    timers,
```

Comment:
This is another one I don't think should be in here, since it would allow `setTimeout`, `setInterval` and those things. Which we can't guarantee the resources for an event would still be valid after the execution has happened and the engine isn't tracking the app. So, I vote to have this one removed since people might mistakenly call it and it is broken.